docs/src/examples/howtorigidtransforms.md

```diff
@@ -0,0 +1,205 @@
+```julia
+# calculate the exponential mapping from point p0
+x1a = ProductRepr(t1a, w1a)   # Lie algebra element
+p1a = exp(G, p0, x1a)         # Lie group element
+
+x1b = ProductRepr(t1b, w1b)
+p1b = exp(G, p0, x1b)
+
+x2 = ProductRepr(t2, w2)
+p2 = exp(G, p0, x2)
+
+x3 = ProductRepr(t3, w3)
+p3 = exp(G, p0, x3)
+
+x4 = ProductRepr(t4, w4)
+p4 = exp(G, p0, x4)
+```
+
+If you want to continue using `ProductRepr` then skip ahead to [Compose Rigid Transforms, 2D](@ref compose_rigid_transforms_2d).
+
+### ProductArray for SpecialEuclidean(2)
+
+As mentioned earlier, there are two data representation formats available for product manifolds.  This paragraph will repeat the same steps as the `ProductRepr` paragraph above, but using the `ProductArray` representation instead.
+
+!!! note
+    `ProductArray` might seem a bit more laborious than `ProductRepr` at first, but the increased structure allows for faster implementation details within the Manifolds.jl ecosystem.
+
+The underlying data is represented according to a shape specification
+```julia
+reshaper = Manifolds.StaticReshaper()
+shape_G = Manifolds.ShapeSpecification(reshaper, M)
+shape_se = Manifolds.ShapeSpecification(reshaper, M.manifolds...)
+```
+
+And the associated Lie group elements (still `SpecialEuclidean(2)`) are mapped from the vector elements above
+```julia
+p0  = Manifolds.prod_point(shape_se, (t0,exp(M_R,  R0, w0))... )
+p1a = Manifolds.prod_point(shape_se, (t1a,exp(M_R, R0, w1a))... )
+p1b = Manifolds.prod_point(shape_se, (t1b,exp(M_R, R0, w1b))... )
+p2  = Manifolds.prod_point(shape_se, (t2,exp(M_R,  R0, w2))... )
+p3  = Manifolds.prod_point(shape_se, (t3,exp(M_R,  R0, w3))... )
+p4  = Manifolds.prod_point(shape_se, (t4,exp(M_R,  R0, w4))... )
+```
+
+Just as with the `ProductRepr` approach, each of the points, e.g. `p2`, represent a relative rigid transform -- note again that the exponential map was done relative to the choosen identity `R0`.  The identity point for translations `M_T` are trivial, i.e. `[0;0]`, and therefore not elaborated as is done with the Rotations under `M_R`.
+
+## [Compose Rigid Transforms, 2D](@id compose_rigid_transforms_2d)
+
+In this section all the details from above are brought together via the action `compose`.  Recall the objective for this tutorial is to walk around a rectangle using the `Manifolds.SpecialEuclidean(2)` mechanizations and abstractions.  Note that either the `ProductRepr` or `ProductArray` representations will dispatch the same for the code below, but the internal data representation formats will differ.
+
+The fine grain details in this section are perhaps not as straight forward as they first appear, specifically how relative transforms (which are points on the manifold `SpecialEuclidean(2)`) relate to the consecutive positions of the walked rectangle (which are also points on the manifold `SpecialEuclidean(2)`).
```

Comment:
You can achieve a similar result using `translate` or `apply` for `GroupOperationAction`. Especially the last one may be helpful in expressing qualitative difference between the elements of SE(2) corresponding to positions, and elements corresponding to transforms.